packages/neon/neon/lib/src/blocs/accounts.dart

```diff
@@ -114,12 +138,19 @@ class AccountsBloc extends Bloc implements AccountsBlocEvents, AccountsBlocState
     }());
   }
 
-  @override
-  void setActiveAccount(final Account? account) {
+  /// Sets the [activeAccount].
+  ///
+  /// Triggers a logout when [account] is null.
+  /// Use [removeAccount] to logout an account.
+  @visibleForTesting
+  void setActiveAccountOrLogout([final Account? account]) {
     final as = accounts.value;
     activeAccount.add(account ?? as.firstOrNull);
   }
```

Comment:
Either I don't understand the point of this function or the documentation is wrong.
This method will just set the active account or if none is give it will use the first available one.
That doesn't mean any account will be logged out right?

I think the whole terms are a bit messy right now, because what means logging out an account? Does it mean it's just no longer the active one or that the account is removed completely?

Reply:
Docs are really bad here 😅